### DISCUSSION TITLE
大佬，有没有三维地名组件啊，类似天地图官网的cesium-tdt

### DISCUSSION THREAD

**Discussion by whyds000** [zouyaoji/vue-cesium#329] [Q&A]
大佬，有没有三维地名组件啊，类似天地图官网的cesium-tdt

**Comment by whyds000** [comment]
我用了下
vc-terrain-provider-tianditu 组件并且指定版本为1.63.1也不行

**Reply by zouyaoji** [reply]
地形嘛？1.92以前的都是可以的。之后的版本TDT没更新，也没开源，我也没办法。

**Reply by whyds000** [reply]
想的用tdt的三维地名标注的功能，之前是叠加了一层高德地图的标注，但是效果不太好

**Reply by zouyaoji** [reply]
没提供 三维地名标注的组件，你只有看他的文档，用js的写法试试。

**Reply by whyds000** [reply]
好的，感谢解答
